docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=password_postgres
       - POSTGRES_DB=aspen_db
+      - PGDATA=/var/lib/czge/data # Avoid using the pgsql image's default volume.
```

Comment:
Nit: I think a bit more explanation here might be nice. It's such a niche thing and kind of unclear why we need to avoid the default volume at a glance. Maybe just toss in a link to the StackOverflow you showed me in case anyone needs to figure out why this is here in two years.